modules/mongodb/manifests/s3backup/restore.pp

```diff
@@ -44,6 +44,8 @@
   $user = 'govuk-backup',
   $cron = false
 ){
+
+  require mongodb::s3backup::package
```

Comment:
This require should be attached to the /usr/local/bin/mongodb-restore-s3 resource to avoid it being 'free floating.'